autoload/test/go/gotest.vim

```diff
@@ -15,6 +19,12 @@ function! test#go#gotest#build_position(type, position) abort
     if a:type ==# 'file'
       return path ==# './.' ? [] : [path . '/...']
     elseif a:type ==# 'nearest'
+       
+      if g:test#go#gotest#testify#enabled == 1 && s:is_testify() == 1
```

Comment:
Is the flag `g:test#go#gotest#testify#enabled` needed when there is the `s:is_testify()` check? It appears it could be removed, making this change require zero configuration, which is the ethos of this plugin

Reply:
I think it should be kept because some developers may mix two way in the same codebase

Reply:
As long as removing it doesn't break any existing tests, I'd rather we remove the need for any additional configuration and only introduce it if and when a need arises. It also keeps the PR simpler as there is no need to update any documentation

Reply:
OK, I've removed the flag to keep it simple